docs/mcp.md

```
@@ -1,11 +1,17 @@
 
-Dofek exposes a remote Model Context Protocol endpoint from the existing API server at:
+Dofek exposes a remote Model Context Protocol endpoint at:
 
 ```text
-https://<your-dofek-host>/api/mcp
+https://dofek.fit/api/mcp
 ```
 
+Production deployments must keep `PUBLIC_URL=https://dofek.fit`; OAuth resource and token audiences are exact canonical URLs, so an alternate production origin is invalid.
+
+### Public-origin cutover precondition
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)